lib/html.rb

```diff
@@ -30,17 +30,17 @@ def self.run_server(system_description_store, opts, &block)
 
         Machinery::Ui.warn <<-EOF.chomp
 Warning:
-The --public option makes the HTTP server listen on all configured IP addresses. Everyone who has access to one of those IP addresses can access all of your system descriptions stored in '~/.machinery'. Be careful if there are sensible information (such as private keys) stored in one of your descriptions.
+The --public option makes the HTTP server listen on all configured IP addresses. Everyone who has access to one of those IP addresses can access all of your system descriptions stored in '~/.machinery'. Be careful if there are sensitive information (such as private keys) stored in one of your descriptions.
```

Comment:
Since we are fixing the gramar let's also change the "are information" to "is information"